tests/tests/tests.py

```diff
@@ -4,9 +4,13 @@
 """
 import itertools
 
+try:
+    from unittest import skipIf, skipUnless
+except ImportError:  # Python < 2.7
+    from django.utils.unittest import skipIf, skipUnless
```

Comment:
why do you just import `skipIf` and `skipUnless`, if you just `import unittest` or `from django.utils import unittest`, you don't have to change the other code in this file.